src/app/index.ts

```diff
@@ -123,6 +136,56 @@ module.exports = class TheiaPlugin extends Base {
             (this.options as any).pluginType = answers.pluginType;
         }
 
+        if (!(this.options as any).template) {
+            const answers = await this.prompt([{
+                type: 'list',
+                name: 'template',
+                message: 'Please, choose the template:',
+                choices: [
+                    {
+                        name: 'Hello World plugin',
+                        value: 'hello-world'
+                    },
+                    {
+                        name: 'Skeleton plugin',
+                        value: 'skeleton'
+                    },
+                    {
+                        name: 'Samples',
+                        value: 'samples'
+                    }
+                ]
+            }]);
+            (this.options as any).template = answers.template;
+        }
+
+        if (!(this.options as any).sample && (this.options as any).template === 'samples') {
+            const answers = await this.prompt([{
+                type: 'list',
+                name: 'sample',
+                message: 'Please, choose the sample:',
+                choices: [
+                    {
+                        name: 'Commands API sample',
+                        value: 'commands'
+                    },
+                    {
+                        name: 'Information message sample',
```

Comment:
is it capitalize or not ? like Information Message sample (for example we have `Quick Pick`)